include/openthread/thread.h

```diff
@@ -97,6 +97,7 @@ typedef struct
     uint8_t      mLinkQualityIn;        ///< Link Quality In
     int8_t       mAverageRssi;          ///< Average RSSI
     int8_t       mLastRssi;             ///< Last observed RSSI
+    uint8_t      mLinkMargin;           ///< Link Margin
```

Comment:
Thanks @Mikeadaneg. Looks good.
You also need to increment API version number in `include/openthread/instnace.h`.

Reply:
@abtink Just updated the API version number to 281